src/category_theory/limits/shapes/bicones.lean

```diff
@@ -0,0 +1,121 @@
+/-
+Copyright (c) 2017 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import category_theory.limits.cones
+import category_theory.structured_arrow
+import category_theory.fin_category
+
+/-!
+# Bicones
```

Comment:
I think it makes more sense to place this file in `category_theory/limits` instead of `category_theory/limits/shapes`.